srcpkgs/guake/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'guake'
+pkgname=guake
+version=3.3.0
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="guake"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel python3-pbr"
+depends="desktop-file-utils libkeybinder3 libnotify python3-dbus python3-gobject
+ python3-pbr vte3"
+checkdepends="python3-flake8 python3-pylint python3-pytest"
+short_desc="Drop-down terminal for GNOME"
+maintainer="John <johnz@posteo.net>"
+license="GPL-2.0-or-later"
+homepage="http://guake-project.org/"
+distfiles="https://github.com/Guake/guake/archive/${version}.tar.gz"
+checksum=36447cd32daf4cb74ac4ebec386d20b659f02e5c7d66315b1078f6da2b894cfd
+
+pre_build() {
+	export PBR_VERSION=4.0.4
+	make generate-paths generate-mo
+}
+
+pre_install() {
+	export PBR_VERSION=4.0.4
+	make prefix=/usr DESTDIR=${DESTDIR} install-locale
+}
+
+post_install() {
+	vmkdir usr/share/applications
+	vmkdir usr/share/pixmaps
+	ln -s /usr/share/guake/data/guake.template.desktop \
+		${DESTDIR}/usr/share/applications/guake.desktop
+	ln -s /usr/share/guake/data/guake-prefs.template.desktop \
+		${DESTDIR}/usr/share/applications/guake-prefs.desktop
+	ln -s /usr/share/guake/data/pixmaps/guake.png \
+		${DESTDIR}/usr/share/pixmaps/guake.png
```

Comment:
Wouldn’t relative links be better? Like e.g.

    ln -s ../guake/data/pixmaps/guake.png ${DESTDIR}/usr/share/pixmaps/guake.png

etc.

Reply:
True. I changed it now to rel. link.